### ISSUE TITLE
Error 23: ) does not have a value. (VariableImporter.jsx)

### ISSUE THREAD

**Issue by Fakepng** [Silly-V/Adobe-Illustrator#66]
# VariableImporter.jsx

```
Error 23: ) does not have a value.
Line: 1
-> ()
```

### I try fixing this with this step:

1. Reinstall illustrator.
2. Redownload VariableImporter.jsx

![image](https://user-images.githubusercontent.com/86796781/209150244-0069ace4-ce63-4870-a78e-218b1a7cfa6e.png)

**Comment by PedroP539** [comment]
It's happening again, and it's driving me MAD.

Can you help, please?

**Comment by hoochalaffa** [comment]
Can you attach the dataset CSV?

**Comment by PedroP539** [comment]
[FloraIberia_FLORES_TEST.csv](https://github.com/user-attachments/files/17589225/FloraIberia_FLORES_TEST.csv)

Yep, attached.

**Comment by hoochalaffa** [comment]
@PedroP539 I was able to use the dataset to merge in the text fields without changing them. I was also able to generate dummy files from the pdf and jpg references and merge those files in without changing them. I opened up your csv with semicolon delimiter and exported it out of excel as a tab-delimited txt file.

when you are on the test tab of the script, do the tests all check out? Are there any errors for dataset names, image files, art bindings?

**Comment by hoochalaffa** [comment]
@PedroP539 looks like this may be solved by deleting the .json settings file created by the script.
https://github.com/Silly-V/Adobe-Illustrator/issues/10

**Comment by PedroP539** [comment]
VariableImporter_SETTINGS.json ?

**Comment by hoochalaffa** [comment]
Based on the link, If the file is empty, it produces that error. The user
deleted it and was able to open the script afterwards.

**Comment by PedroP539** [comment]
Hi Jonathan,

Yes, i did! It's working fine now. Your help was invaluable!

**Comment by hoochalaffa** [comment]
@PedroP539 Sorry! I didn't see your previous reply. Glad you had it figured out!! (hopefully it was in time to meet your deadline.)

lol, if my wife ever gets me out of the country to a vacation, Portugal is on her list. I'll drop a line to you when that happens.

**Comment by PedroP539** [comment]
Hi dear friends,

It's happening again, but now it's really strange, as the VariableImporter_SETTINGS.json is empty (after creating a Preset) and pops out the same error when trying to import a CSV file.

If i delete the json file, it works, but blows away any preset created. This is not practical and implies a lot of manual work.

If i try to open a different file - another CSV - , and use Variable Importer, it pops out:

Error 23: ) does not have a value.
Line: 1
-> ()

TLDR: It's impossible to have multiple Presets with these errors, and constantly deleting the json file only aloows me a single preset.

haaaaalp! ⛑
